vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -466,6 +490,68 @@ and
 
         setup (siteProvider.GetProjectSite()) |> ignore
 
+    member private __.ProvideProjectSiteProvider(workspace:Workspace, project:Project) =
+        let visualStudioWorkspace = workspace :?> VisualStudioWorkspace
+        let hier = visualStudioWorkspace.GetHierarchy(project.Id)
+        {new IProvideProjectSite with
+             member this.GetProjectSite() =
+                 let compileItems () = [| for document in project.Documents do yield document.FilePath |]
+                 let compilerFlags () = [| "" |]
+                 let caption () = project.Name
+                 let projFileName () = project.FilePath
+                 let taskProvider = None
+                 let taskReporter = None
+                 let targetFrameworkMoniker = ""
+                 let projectGuid () = project.Id.Id.ToString()
+                 let creationTime = System.DateTime.Now
```

Comment:
Consider using UTC - this could go backwards if VS is running when the timezone/DST of the machine changes.